pod-operation/src/components/lim_current.rs

```diff
@@ -0,0 +1,52 @@
+use ads1x1x::ic::{Ads1015, Resolution12Bit};
+use ads1x1x::interface::I2cInterface;
+use ads1x1x::mode::OneShot;
+use ads1x1x::ChannelSelection::{SingleA0, SingleA1, SingleA2};
+use ads1x1x::{Ads1x1x, SlaveAddr, FullScaleRange};
+use nb::block;
+use rppal::i2c::I2c;
+
+fn voltage_to_current(voltage: i16) -> f32 {
+
+    let SLOPE = 15.873;
+    let YINTERCEPT = 39.683;
```

Comment:
Chore: these constants could be extracted to the global scope and declared `const`.
Issue: these names still don't have any particular relevance, could we rename them to what they actually physically/electronically signify and/or derive them from more primitive values?

Reply:
We referenced a graph which was a linear function to find slope and y-intercept. We then used the coordinates of 2 points to find these values. I don't think there is a descriptive name for these values.

Reply:
![Screenshot from 2024-05-21 14-12-20](https://github.com/UCI-HyperXite/hx9-control-system/assets/103944547/e2e71105-8eeb-4173-94fd-59f446ea3136)

Reply:
Let's strive for clear intent and formulation then: we can define `QUIESCENT_VOLTAGE` as 2.5 V and `SENSITIVITY` as 66 mV/A (or 0.066 V/A) to specify the conversion as
```rust
current = (voltage - QUIESCENT_VOLTAGE) / SENSITIVITY
```